2/homework/laughter_classification/sspnet_data_sampler.py

```diff
@@ -88,8 +92,12 @@ def create_sampled_df(self, frame_sec, naudio=None, save_path=None, force_save=F
         :param force_save: boolean, if you want to override file with same name
         :return:
         """
+        if save_path is not None and os.path.isfile(save_path):
```

Comment:
I noticed you use `os.path.isfile` often
As a general suggestion take a loot at pathlib: I prefer it over os.path in general and one of a features if that `Path` object checks file existence automatically